packages/portfolio/src/adapters/dullahan-api/api-maker.ts

```diff
@@ -336,6 +366,8 @@ export const apiConfig: ApiConfig = freeze(
         'https://yoroi-backend-zero-sanchonet.emurgornd.com/tokens/activity/multi',
       tokenPriceHistory:
         'https://add50d9d-76d7-47b7-b17f-e34021f63a02.mock.pstmn.io/v1/token-price-history',
+      tokenImageInvalidate:
+        'https://preprod.processed-media.yoroiwallet.com/invalidate',
```

Comment:
sancho is about to die, anyways.
```suggestion
        'https://sancho.processed-media.yoroiwallet.com/invalidate',
```

Reply:
That api doesn't have a `sancho` endpoint